src/plays/markdown-editor/Downloader.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+// html2canvas for converting the md to png image
+import html2canvas from 'html2canvas';
```

Comment:
Lets not use `html2canvas` as it brings the style as well

Reply:
![image](https://user-images.githubusercontent.com/7145967/183733853-b1497e21-c2bb-4e22-ae5a-7d612ca1fc35.png)